src/routers/genericRouters/routerFuzzy.ts

```diff
@@ -19,28 +19,55 @@ export class RouterFuzzy extends RouterFilterBy implements Router {
     return `${this.dbCollectionName}_fuzzy_search_alias`
   }
 
-  async getObjectsByFuzzyTextSearch (term: string, page: number, customFilter: string = ''): Promise<any[]> {
-    // supporting only one search field for now
-    let searchField = this.fuzzyTextSearch[0]
+  levenshtein (searchField: string, searchTerm: string): string {
```

Comment:
Can you add comment doc for those three methods: levenshtein, multipleToken and autocomplete. Thank youQ

Reply:
Sure! Added! Let me know if I should extend it